src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  margin: 0;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__items {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__icon {
+    display: flex;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  // &--2 {
+  //   margin-left: 57px;
+  // }
+
+  &__title {
+    display: flex;
+    width: 166px;
```

Comment:
Consider avoiding hardcoding width. Setting padding for the parent block should be enough for this task